services/brig/src/Brig/User/Event/Log.hs

```diff
@@ -39,3 +40,6 @@ instance ToBytes PropertyEvent where
 instance ToBytes ClientEvent where
     bytes (ClientAdded u _)   = val "user.client-add: " +++ toByteString u
     bytes (ClientRemoved u _) = val "user.client-remove: " +++ toByteString u
+    bytes (LegalHoldClientRequested payload) =
+      val "user.client-request-legal-hold: " +++ show payload
```

Comment:
are you sure you want to use `show` here?  `toByteString` renders `Id a` as `"93074dc8-8174-11e9-a76b-1361075da847"`, which is more concise.  (not sure whom or what we do this for here...)

Reply:
This is just for logging so I just did the "quick thing"™

If you feel strongly I can do something nice, but I think 'show' is sufficient for debugging 🤷‍♂